mep/settings.py

```diff
@@ -155,6 +155,8 @@
 
 USE_TZ = True
 
+# Use three-letter month names everywhere (instead of default AP style)
+DATE_FORMAT = 'M j, Y'
```

Comment:
nice, good call putting this here

Reply:
Came out of a very timely conversation with @gissoo when we were looking at the card list designs and implementation together!